microsoft-edge/web-platform/os-regional-settings.md

```diff
@@ -0,0 +1,88 @@
+
+A regional code is a hyphenated 2 letter language code followed by a 2 letter region code (generally—the standard for language and regional codes is quite complex: [chromium uses ICU an implementation of the Unicode Standard’s Internationalization API](https://icu.unicode.org/home). See their docs for specific details.) For example: “en-US” means the United States region where English is the primary language. “en-GB” means English with the preferences of Great Britain (UK). Similarly, other language and regional codes express these pairs for other languages: “pt-BR” means Brazilian Portuguese. 
+
+To setup the user’s language preferences, Edge gets the default values from the OS at the time Edge is first installed. After that, users can change the language preferences via edge://settings/languages, including adding multiple languages (with or without region information) and sorting them in a priority order. That language information is used in a variety of ways:
+
+|   | What Setting Edge uses? |
+|---|-------------------------|
+| Language used for rendering Edge’s UI | edge://settings/languages. One item in the “Languages” list will indicate “Microsoft Edge is displayed in this language” |
+| Language used to render web sites | **Website Server-side processing**: Edge sends the language list (as regional codes) specified in edge://settings/languages via the accept-language header. <br><br> **Website Client-side processing**: Edge makes the language list (as regional codes) available via the navigator.languages DOM API. The preferred language (at the top of the list) is also available as navigator.language (note: singular) |
+| Language/region used for formatting date, time, numbers, etc. | The language in the edge://settings/languages list specified as the “displayed language” |
+
+While many OSes supported by Microsoft Edge do have a facility to specify regional information preferences, this OS information (as a regional code) is not used by the browser, and customers who make these changes to the OS have no similar mechanism to express these regional preferences in Microsoft Edge (except to change their display language and region). 
+
+## Opportunity
+In order to better reflect a customer’s regional preferences (especially in the cases where the customer has changed their default preferences in the OS), Microsoft Edge is making a new Setting available in the “Language” section of Settings:
+
+![Screenshot of Edge setting UI titled "Share additional OS regional format" with the option "limited (default)" selected.](media/os-regional-setting-screenshot.msft.png)
+
+This setting allows the customer’s OS regional preferences to be automatically made available to websites under “limited” cases by default (more on that below), and offers two other options, including “never” which reverts the feature back to legacy behavior. 
```

Comment:
at "below" I'll try to link to a h2 or h4 heading

Reply:
I just removed the line because it's literally below and not needed with the new headers